tsdb/head.go

```diff
@@ -112,6 +112,7 @@ type headMetrics struct {
 	outOfOrderSamples        prometheus.Counter
 	walTruncateDuration      prometheus.Summary
 	walCorruptionsTotal      prometheus.Counter
+	walTotalReplayDuration   prometheus.Gauge
```

Comment:
Should this be a `Summary`? Though it's observed only once. @roidelapluie

Reply:
There is only one observation, quantiles are not useful with one observation.

Reply:
Then gauge sounds fine. I was not sure about the usage :)